convert-to-pdf.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+npx mr-pdf --initialDocURLs="http://localhost:3000" --contentSelector="article" --paginationSelector=".pagination-nav__item--next > a" --excludeSelectors=".margin-vert--xl a,.theme-edit-this-page,.tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh,.tocCollapsibleButton_2O1e" --coverImage="https://download.logo.wine/logo/Seneca_College/Seneca_College-Logo.wine.png" --coverTitle="IPC144" --outputPDFFilename "Seneca-IPC144.pdf"
```

Comment:
Add a newline at the end of this line (e.g., press enter so it goes to the next line).  Don't end a file on the same line.